tensorflow/lite/kernels/internal/reference/strided_slice.h

```diff
@@ -18,74 +18,51 @@ limitations under the License.
 #include "tensorflow/lite/kernels/internal/common.h"
 #include "tensorflow/lite/kernels/internal/strided_slice_logic.h"
 #include "tensorflow/lite/kernels/internal/types.h"
+#include <functional>
+
 namespace tflite {
 
 namespace reference_ops {
 template <typename T>
 inline void StridedSlice(const tflite::StridedSliceParams& op_params,
-                         const RuntimeShape& unextended_input_shape,
-                         const T* input_data,
-                         const RuntimeShape& unextended_output_shape,
-                         T* output_data) {
-  using strided_slice::LoopCondition;
-  using strided_slice::StartForAxis;
-  using strided_slice::StopForAxis;
-  // Note that the output_shape is not used herein.
-  tflite::StridedSliceParams params_copy = op_params;
+						 const RuntimeShape& input_shape,
+						 const T* input_data,
+						 const RuntimeShape& output_shape,
+						 T* output_data) {
+  const int slice_dimensions = input_shape.DimensionsCount();
+  T* out_ptr = output_data;
 
-  TFLITE_DCHECK_LE(unextended_input_shape.DimensionsCount(), 5);
-  TFLITE_DCHECK_LE(unextended_output_shape.DimensionsCount(), 5);
-  const RuntimeShape input_shape =
-      RuntimeShape::ExtendedShape(5, unextended_input_shape);
-  const RuntimeShape output_shape =
-      RuntimeShape::ExtendedShape(5, unextended_output_shape);
+  std::vector<int> start;
```

Comment:
TFLite Micro cannot allocate memory dynamically so vector should be avoid.

Reply:
@thaink : Thanks for review! So how much we should allocate ?